include/flang/Error/errmsg-in.n

```diff
@@ -1334,8 +1334,9 @@ LAUNCH_BOUNDS is now allowed on ATTRIBUTES(DEVICE) or host subprograms.
 The LAUNCH_BOUNDS maximum number of threads and minimum grid size must
 be positive integer values.
 .MS W 602 "No clause specified for the vector directive. Note: Only the always clause is supported."
-.MS W 603 "Unsupported clause specified for the vector directive. Only the always clause is supported."
+.MS W 603 "Unsupported clause specified for the vector directive. Only the always and vectorlength clause is supported."
```

Comment:
There is a problem with the grammar here. I suggest:
```
Only the always and vectorlength clauses are supported.
```

Reply:
@bryanpkc fixed